Pipelines/Config/settings.yaml

```diff
@@ -0,0 +1,8 @@
+variables:
+  # Note that when updating this value also ensure that you update the
+  # following locations to match. There will be a CI failure if they don't
+  # match (see scripts/packaging/versionmetadata.ps1)
+  # ProjectSettings/ProjectSettings.asset:  bundleVersion: x.x.x
```

Comment:
Could you explain this requirement? Do we need to update the bundleVersion and metroPackageVersion when we change the MRTKGraphicsToolsVersion?